packages/jest-config/index.js

```diff
@@ -15,4 +15,5 @@ module.exports = {
     "^~/(.*)$": "<rootDir>/src/$1",
   },
   extensionsToTreatAsEsm: [".ts", ".tsx"],
+  prettierPath: null,
```

Comment:
Jest seem does not support prettier v3 because it removed synchronous api. Jest seem less suitable for us now.